valinta-tulos-valintarekisteri-db/src/main/scala/fi/vm/sade/valintatulosservice/valintarekisteri/db/impl/SijoitteluRepositoryImpl.scala

```diff
@@ -413,6 +413,15 @@ trait SijoitteluRepositoryImpl extends SijoitteluRepository with Valintarekister
               where hakijaryhma_oid = ${hakijaryhmaOid} and sijoitteluajo_id = ${sijoitteluajoId} and hyvaksytty_hakijaryhmasta = TRUE""".as[HakemusOid]).toList
     }
 
+  override def getHaunSijoittelemattomatHakukohteet(hakuOid: HakuOid): Set[HakukohdeOid] = {
+    runBlocking(sql"""select distinct hk.hakukohde_oid
```

Comment:
loggausta tähän kohtaan?
```suggestion
    logger.debug(s"Fetching sijoittelemattomat hakukohteet for hakuOid: $hakuOid")
    runBlocking(sql"""select distinct hk.hakukohde_oid
```